six/modules/c++/cphd/include/cphd/CPHDWriter.h

```diff
@@ -84,11 +84,15 @@ class CPHDWriter
      *         data in chunks. Otherwise you should use addImage and write.
      *         This will internally set the number of bytes per VBP.
      *
-     *  \pathname The desired pathname of the file.
-     *  \vbm The vector based metadata to write.
+     *  \param pathname The desired pathname of the file.
+     *  \param vbm The vector based metadata to write.
+     *  \param classification The classification of the file
```

Comment:
Add a note to these comments and the other ones that these two fields are optional and by default the CPHD will not be populated with these values